tools/docker_dev/.gitpod.Dockerfile

```diff
@@ -0,0 +1,79 @@
+# Using the Scipy-dev Docker image as a base
+# This way, we ensure we have all the needed compilers and dependencies
+# while reducing the build time
+ARG BASE_CONTAINER=trallard/scipy:scipy-dev.4
```

Comment:
pinning now - will add a script to get the last updated and tagged image from `scipy`